src/model/get-order.model.ts

```diff
@@ -0,0 +1,60 @@
+import { IRetrieveOrder, IPayload, IRetrieveInfo, IRetrieveItem } from '../interface';
+import { ServiceTypeEnum, PaymentTypeEnum } from '../enum';
+import { Customer } from './create-order.model';
+
+export class RetrieveOrder implements IRetrieveOrder {
+    order: {
+        guid: string;
+        payload: Payload;
+        external_id: string;
+        created_at: string;
+        completed_at: string | null;
+        error_at: string | null;
+        error_string: string | null;
+        ticket_payload: null;
+    };
+}
+
+export class Payload implements IPayload {
+    customer: Customer;
+    info: RetrieveInfo;
+    items: RetrieveItems[];
+    configuration: {};
+}
+
+export class RetrieveInfo implements IRetrieveInfo {
```

Comment:
looks like `RetrieveInfo` class is useless (if you need it only for plainToClass needs - maybe makes sense to have only class and remove the interface)

Reply:
think about it

Reply:
I think is need, Is used in get-order.model and fixture to declare properties.

Reply:
if you have 
```
interface IA { name: string };

class A implements IA { name: string };

const fixture: IA = { name: "John" } === const fixture: A = { name: "John" }
```